### PR TITLE
[21698] Add -v or --version to cli command tool

### PR DESCRIPTION
**Pull request by depink5**
Modify the cli command tool to provide the version of the installed Fast DDS. Modify CMakeLists.txt in fastdds repository to export 'package.xml' to install directory.

## Contributor Checklist

- [X] Commit messages follow the project guidelines. 
- [X] The code follows the style guidelines of this project. 
- [X] Tests that thoroughly check the new feature have been added/Regression tests checking the bug and its fix have been added; the added tests pass locally 
- _N/A_ Any new/modified methods have been properly documented using Doxygen. 
- _N/A_ Any new configuration API has an equivalent XML API (with the corresponding XSD extension) 
- _N/A_ Changes are backport compatible: they do **NOT** break ABI nor change library core behavior. 
- _N/A_ Changes are API compatible. 
- _N/A_ New feature has been added to the `versions.md` file (if applicable).
- _N/A_ New feature has been documented/Current behavior is correctly described in the documentation. 
    
- _N/A_ Applicable backports have been included in the description.

## Reviewer Checklist

- [X] The PR has a milestone assigned.
- [X] The title and description correctly express the PR's purpose.
- [X] Check contributor checklist is correct.
- _N/A_ If this is a critical bug fix, backports to the critical-only supported branches have been requested.
- [x] Check CI results: changes do not issue any warning.
- [x] Check CI results: failing tests are unrelated with the changes.

### CODE REVIEW

**Review by cferreiragonz — CHANGES_REQUESTED** [review]
Looks great so far! Just a few suggestions:

- I would add one test to check this behavior to `test/system/tools/fastdds/tests.py`. In this way we can test it works in other platforms and that future commits don't break the new feature.
- I would change the indentation in the `package.xml` file to avoid printing the description with indentation. Also we can change `eprosima` to `eProsima`.
- Finally, we could try to parse the description to use shorter sentences. This will improve readability in small terminals. This is just an aesthetic suggestion though.

File: tools/fds/server.h [hunk 1]
```diff
@@ -61,6 +62,9 @@ const option::Descriptor usage[] = {
 
     { HELP,      0, "h",  "help",         Arg::None,
       "  -h  \t--help        Produce help message.\n" },
+    
+    { VERSION,   0, "v",  "version",      Arg::None,
+      "  -v  \t--version     Show FastDDS version information.\n" },
```
Comment (MiguelCompany) [inline tools/fds/server.h:67]:
```suggestion
      "  -v  \t--version     Show Fast DDS version information.\n" },
```

File: tools/fastdds/discovery/parser.py [hunk 2]
```diff
@@ -54,6 +54,14 @@ def __init__(self, argv):
                     (len(argv) == 1 and argv[0] == '--help')
                ):
                 print(self.__edit_tool_help(result.stdout))
+
```
Comment (cferreiragonz) [inline tools/fastdds/discovery/parser.py:57]:
```suggestion
```

File: test/system/tools/fastdds/tests.py [hunk 3]
```diff
@@ -96,6 +96,13 @@ def test_fastdds_installed(install_path):
         print('test_fastdds_installed FAILED')
         sys.exit(ret)
 
+def test_fastdds_version(install_path):
+    """Test that fastdds version is printed correctly."""
+    args = ' -v'
+    ret = subprocess.call(cmd(install_path, args=args), shell=True)
+    if 0 != ret:
+        print('test_fastdds_version FAILED')
+        sys.exit(ret)
```
Comment (cferreiragonz) [inline test/system/tools/fastdds/tests.py:105]: We need to add the test to the `test/system/tools/fastdds/CMakeLists.txt` in order to build it and be able to run it. Remember also to add it to the `Tests dictionary` at the end of the file.

File: tools/fds/server.cpp [hunk 4]
```diff
@@ -109,6 +110,13 @@ int fastdds_discovery_server(
         return 0;
     }
 
+    // Show version if asked to
+    if (options[VERSION])
+    {
+        std::cout <<  "Fast DDS version: " << FASTDDS_VERSION_STR << std::endl;
```
Comment (cferreiragonz) [inline tools/fds/server.cpp:116]:
```suggestion
        std::cout << "Fast DDS version: " << FASTDDS_VERSION_STR << std::endl;
```

File: tools/fastdds/fastdds.py [hunk 5]
```diff
@@ -42,7 +42,10 @@
 """
 
 import argparse
+import pathlib
 import sys
+import textwrap
+import xml.etree.ElementTree as ET
```
Comment (cferreiragonz) [inline tools/fastdds/fastdds.py:48]:
```suggestion
import sys
```
We are not using these libraries anymore.

**Review by cferreiragonz — CHANGES_REQUESTED** [review]

**Review by cferreiragonz — APPROVED** [review]
LGTM with green CI

**Pull request merged**